source/javascripts/_input.es6

```diff
@@ -27,5 +27,17 @@ $(function() {
     });
   }
 
+  $.fn.radioify = function() {
+    let $elements = $(this);
+
+    $elements.on('focusin', function(e) {
+      let $radioButton = $(e.target);
+      if (!$radioButton.filter(':checked').val()) {
```

Comment:
What about `$radioButton.find(':checked').length == 0` or something like that? Not sure what reads better, or is more idiomatic with jQuery.